<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WinWidget</class>
 <widget class="QWidget" name="WinWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="cover" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>470</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(0, 0, 0, 220)</string>
   </property>
   <widget class="QLabel" name="moveCount">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>230</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;Menlo&quot;;
qproperty-alignment: AlignRight;
background: none;
color: #E1E3E5;</string>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="randomButton">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>350</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FontAwesome</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	color: #A1A3A5;
	background: none;
	border: none;
	font: 20pt &quot;FontAwesome&quot;;
}

QPushButton:hover
{
	color: #C1C3C5;
}

QPushButton:pressed
{
	color: #404244;
	background: #C1C3C5;
}</string>
    </property>
    <property name="text">
     <string></string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="moves">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>230</y>
      <width>201</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Menlo&quot;;
qproperty-alignment: AlignRight;
background: none;
color: #E1E3E5;</string>
    </property>
    <property name="text">
     <string>Moves:</string>
    </property>
   </widget>
   <widget class="QLabel" name="best">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>280</y>
      <width>191</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;Menlo&quot;;
qproperty-alignment: AlignRight;
background: none;
color: #E1E3E5;</string>
    </property>
    <property name="text">
     <string>Best:</string>
    </property>
   </widget>
   <widget class="QLabel" name="missionComplete">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>79</y>
      <width>400</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 36pt &quot;Menlo&quot;;
qproperty-alignment: AlignCenter;
background: none;
color: #E1E3E5;</string>
    </property>
    <property name="text">
     <string>Mission
Complete</string>
    </property>
   </widget>
   <widget class="QLabel" name="bestCount">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>280</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;Menlo&quot;;
qproperty-alignment: AlignRight;
background: none;
color: #E1E3E5;</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>350</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FontAwesome</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	color: #A1A3A5;
	background: none;
	border: none;
	font: 20pt &quot;FontAwesome&quot;;
}

QPushButton:hover
{
	color: #C1C3C5;
}

QPushButton:pressed
{
	color: #404244;
	background: #C1C3C5;
}</string>
    </property>
    <property name="text">
     <string></string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="star_perfect">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>301</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(234, 136, 0, 40);
background-color: none;
font: 200pt &quot;FontAwesome&quot;;
border: none;</string>
    </property>
    <property name="text">
     <string></string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="restartButton">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>350</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>FontAwesome</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	color: #A1A3A5;
	background: none;
	border: none;
	font: 20pt &quot;FontAwesome&quot;;
}

QPushButton:hover
{
	color: #C1C3C5;
}

QPushButton:pressed
{
	color: #404244;
	background: #C1C3C5;
}</string>
    </property>
    <property name="text">
     <string></string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>star_perfect</zorder>
   <zorder>moveCount</zorder>
   <zorder>randomButton</zorder>
   <zorder>moves</zorder>
   <zorder>best</zorder>
   <zorder>bestCount</zorder>
   <zorder>nextButton</zorder>
   <zorder>missionComplete</zorder>
   <zorder>restartButton</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
